sentry-spring/src/test/kotlin/io/sentry/spring/reactive/SentryReactiveWebFilterTest.kt

```diff
@@ -0,0 +1,82 @@
+package io.sentry.spring.reactive
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.check
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import io.sentry.Breadcrumb
+import io.sentry.IHub
+import io.sentry.SentryOptions
+import javax.servlet.ServletRequestEvent
+import kotlin.test.Test
+import org.assertj.core.api.Assertions
+import org.springframework.mock.http.server.reactive.MockServerHttpRequest
+import org.springframework.mock.web.server.MockServerWebExchange
+import org.springframework.web.server.WebFilterChain
+import reactor.core.publisher.Mono
+import reactor.test.StepVerifier
+
+class SentryReactiveWebFilterTest {
+
+    private class Fixture {
+        val baseHub = mock<IHub>()
+        val hub = mock<IHub>()
+        val userProvider = mock<SentryReactiveUserProvider>()
+        val filter = SentryReactiveWebFilter(baseHub, SentryOptions())
```

Comment:
I'd create a `getSut` for the `SentryReactiveWebFilter` instance, so each test creates a new instance of it, a better isolation test.

Reply:
Thanks @marandaneto, I just made sure that a new instance of the Fixture class (which contains an instance of SentryReactiveWebFilter) is being created for every test in the file, so that we can guarantee tests isolation.